acceptance-tests.md

```diff
@@ -89,9 +106,10 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Online Documentation**
     * _As an administrator or a user, I should be able to perform any routine operations by referring to online documentation which lists the steps that need to be taken to complete an operation._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. Administrator/user accesses documentation at:
             1. [https://nerc-project.github.io/nerc-docs/](https://nerc-project.github.io/nerc-docs/)
+    * Acceptance tests: requires investigation
```

Comment:
Just requires listing the locations of all ops docs.  Besides NERC repo you have listed, there is Red Hat support documentation for what we installed (ACM, OpenShift etc), and the documentation and code for what was done with ColdFront and XDMOD.  You will have to ask about those two at the NERC meeting.  So acceptance test is just verifying that each part of the NERC builld is covered by the documentation links in this section and the links are live.  You may have to say something about credentials too, since Red Hat stuff sometimes requires a license.

Reply:
[ColdFront Upstream](https://github.com/ubccr/coldfront/blob/master/README.md)
[What is NERC's ColdFront](https://github.com/nerc-project/nerc-docs/blob/main/docs/get-started/get-an-allocation.md)

[XDMoD Upstream](https://github.com/ubccr/xdmod/blob/xdmod10.5/README.md)
[MOC XDMoD Readme](https://github.com/CCI-MOC/xdmod-cntr/blob/main/README.md)
[NERC XDMoD OpenShift Integration](https://github.com/OCP-on-NERC/docs/blob/main/xdmod_integration.md)

Reply:
@heidi I will work with @jtriley to ensure this list gets into their runbook.
Hello @jtriley the list we talked about in the meeting today is [2 comments up](https://github.com/OCP-on-NERC/docs/pull/37#discussion_r1024416780). Please let me know how I can help ensure they get where they need to be. If the runbook is in a location I can update I would be happy to create a PR to update it.